examples/resnet/README.md

```diff
@@ -15,20 +15,20 @@ The workflow in [resnet_static_ptq_cpu.json](resnet_static_ptq_cpu.json) is simi
 The workflow in [resnet_dynamic_ptq_cpu.json](resnet_dynamic_ptq_cpu.json) is similar to the above workflow, but specifically uses dynamic quantization instead of static/dynamic quantization.
 
 
-### ResNet optimization with Vitis-AI PTQ on CPU
-This workflow performs ResNet optimization on CPU with AMD Vitis-AI Quantization. It performs the optimization pipeline:
+### ResNet optimization with Vitis-AI PTQ on DPU

```

Comment:
I think it should only be this line? the others should still be CPU.

Reply:
good catch